### PR TITLE
'NameFrame' class added

### PR DESCRIPTION
**Pull request by MadayAlcala**
Hi guys, in this branch attached the portable VLC, please check it out.

### CODE REVIEW

File: src/main/java/com/fundation/logic/view/NameFrame.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+public class NameFrame extends JFrame {
+    private CustomLabel textName;
+    private CustomTextField textFieldName;
+    private CustomButton btnSave;
+
+
+    /**
+     * Inits details frame.
+     *
+     * @param - Path of the file to show details from
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/NameFrame.java]: deleted param

File: src/main/java/com/fundation/logic/view/NameFrame.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        namePanel.add(textName);
+        namePanel.add(textFieldName);
+        namePanel.add(btnSave);
+        add(namePanel, BorderLayout.CENTER);
+        setVisible(true);
+    }
+
+    /**
+     * This method return the content of TextName.
+     *
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/NameFrame.java]: delete unnecessary line

File: src/main/java/com/fundation/logic/view/NameFrame.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+     * @return TextName.
+     */
+    public JLabel getTextName() {
+        return textName;
+    }
+
+    /**
+     * This method return the content of textFieldName.
+     *
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/NameFrame.java]: delete unnecessary line

File: src/main/java/com/fundation/logic/view/NameFrame.java (hunk 1 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/NameFrame.java]: Solved.

File: src/main/java/com/fundation/logic/view/NameFrame.java (hunk 2 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/NameFrame.java]: Solved.

File: src/main/java/com/fundation/logic/view/NameFrame.java (hunk 3 above)
Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/NameFrame.java]: Solved.

File: src/main/java/com/fundation/logic/view/NameFrame.java [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+ *
+ * @author Maday Alcalá
+ * @version 1.0
+ */
+public class NameFrame extends JFrame {
+    private CustomLabel textName;
+    private CustomTextField textFieldName;
+    private CustomButton btnSave;
+
+
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/view/NameFrame.java]: delete unnecessary empty line

Comment (MadayAlcala) [reply src/main/java/com/fundation/logic/view/NameFrame.java]: Solved.

**Pull request merged**